server/deployster_service.go

```diff
@@ -83,3 +86,24 @@ func (ds *DeploysterService) authenticated(h http.Handler) tigertonic.FirstHandl
 		"Deployster",
 		h)
 }
+
+func getFleetHTTPClient() (client.API, error) {
```

Comment:
This is a lot of ugly copy/paste from `github.com/coreos/fleet/fleetctl`.  This should get cleaned up a bit before merge.